code/src/App.js

```diff
@@ -1,12 +1,38 @@
 import React from 'react'
 import data from './data.json'
+import Album from './components/Album'
 
-console.log(data)
 
 export const App = () => {
+
+// AN ARRAY WITH ALL ALBUMS
+const albumArray = data.albums.items
+
+
   return (
-    <div>
-      Find me in src/app.js!
+    <>
+    <div className="main">
+      <div className="header-container">
+        <h1 className="heading">New albums & singles</h1>
+      </div>
+      <div className="album-wrapper">
+
+        {albumArray.map((album) => {
+              return (
+              <Album 
+              image={album.images[0].url} 
+              title={album.name} 
+              hrefAlbum={album.external_urls.spotify} 
+              artist={album.artists.map((item, index) => <a>{item.name}</a>)}
+              hrefArtist={album.artists.map((item) => item.external_urls.spotify)}
+              key={album.id}/>
+             )
+          })
+        }
+      </div>
     </div>
+    </>
   )
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
        <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
	<>
          <div>My element 1</div>
	  <div>My element 2</div>
	</>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
);
```